### PR TITLE
Update index allure report support.js

### LINKED ISSUE #10: Integrate Allure Reporting
add allure, reporter support, check screenshotOnFail.js plugin

### PR DESCRIPTION
**Pull request by srbhss**
Fixes #10 
Idea about Integrating allure report.
Please review it @puneet0191

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -222,6 +224,12 @@ class ResembleHelper extends Helper {
         return boundingBox;
     }
 
+    {
+   const allureReporter = Container.plugins('allure');
+        if (allureReporter) {
+          allureReporter.addAttachment('Screenshot diffImage', fs.readFileSync(path.join(global.output_dir, fileName)), 'image/png');
```
Comment (puneet0191) [inline index.js]: Use the actual Diff Image name, using Test Names

Comment (srbhss) [reply index.js]: Tried.

File: index.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ const fs = require('fs');
 let assert = require('assert');
 const mkdirp = require('mkdirp');
 const getDirName = require('path').dirname;
+const Container = require('../container');
+const Codeceptjs = require('../index');
```
Comment (puneet0191) [inline index.js]: why you need this?

Comment (srbhss) [reply index.js]: Because we need container module for allure report. Updated my code again please review

Comment (srbhss) [reply index.js]: @puneet0191 updated it now please see

**Pull request closed without merging**